core/settings.py

```diff
@@ -31,8 +31,9 @@
     'binary/octet-stream',
 ]
 HMAC_KEY = socket.gethostname() + 'set_0n3_pl3@s3'
-HOST = '0.0.0.0'
-PORT = 9090
-DEBUG = False
+HOST = os.environ.get('HOST', '0.0.0.0')
+PORT = os.environ.get('PORT', 9090)
+DEBUG = os.environ.get('DEBUG', True)
```

Comment:
I would suggest
```
HOST = os.environ.get('NJS_HOST', '0.0.0.0')
PORT = os.environ.get('NJS_PORT', 9090)
DEBUG = os.environ.get('NJS_DEBUG', True)
``
to avoid conflicts!

Reply:
Debug should be false by default

Reply:
Thanks, @ajinabraham for pointing me :) will take a look to production server